Product Designer/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+       <title>
+            Product Designer
+       </title>
+       <link rel="stylesheet" href="/Product Designer/assets/css/styles.css" />
+    </head>
+        <body>
+        <!-- START : HEADER SECTION -->
+        <header class="mailSection">
+            <h6>shruthi.bhatt.siriuscom.com</h6>
+        </header>
+        <!-- END :  HEADER SECTION -->
+
+        <!-- START : NAME SECTION -->
+        <section class="nameSection">
+            <h1 class="name">
+                <span>Shruthi</span> Bhatt
+            </h1>
+            <h2 class="role">
+                a <span>product</span> designer
+                <h6 class="mobileEmail">shruthi.bhatt.siriuscom.com</h6>
+            </h2>
+        </section>
+        <!-- END : NAME SECTION -->
+
+        <!-- START : PRODUCTS SECTION -->
+        <div class="productsContainer">
+            <img src="/Product Designer/assets/Images/rawpixel-769308-unsplash.png" alt="Product Image" />
```

Comment:
use `./` to start the image src for relative path

Reply:
ADDED RELATIVE PATH FOR IMAGES